LoopKitUI/InsulinKit/InsulinDeliveryTableViewController.swift

```diff
@@ -475,3 +492,48 @@ fileprivate extension UIAlertController {
         ))
     }
 }
+
+extension DoseEntry {
+
+    fileprivate var numberFormatter: NumberFormatter {
+        let numberFormatter = NumberFormatter()
+        numberFormatter.maximumFractionDigits = DoseEntry.unitsPerHour.maxFractionDigits
+        return numberFormatter
+    }
+
+    fileprivate var localizedAttributedDescription: NSAttributedString? {
+        let font = UIFont.preferredFont(forTextStyle: .body)
+
+        switch type {
+        case .basal, .bolus, .tempBasal:
+            let unitString = type == .bolus ? DoseEntry.units.shortLocalizedUnitString() : DoseEntry.unitsPerHour.shortLocalizedUnitString()
+            let value: Double = type == .bolus ? (deliveredUnits ?? programmedUnits) : unitsPerHour
+
+            let description = String(format: NSLocalizedString("%1$@: <b>%2$@</b> %3$@", comment: "Description of a basal, bolus, or temp basal dose entry (1: title for dose type, 2: value (? if no value) in bold, 3: unit)"), type.localizedDescription, numberFormatter.string(from: value) ?? "?", unitString)
+            let descriptionWithFont = String(format:"<style>body{font-family: '-apple-system', '\(font.fontName)'; font-size: \(font.pointSize);}</style>%@", description)
```

Comment:
[nit] I think the above usage of "<b>...</b>" is simple and straightforward enough, but this line seems overly complicated. Can we use the first line and then wrap with the previous attributed text code to wrap it with the font settings? After all, the font settings apply to all of the text while the bold only applies to the value.

Reply:
I think I understand you question. Unfortunately, applying the font using the typical methods overrides the `bold` tag.